### PR TITLE
💄 Fixes the overflowing download dropdown and refactors some styles

### PR DESCRIPTION
**Pull request by hlminh2000**

### CODE REVIEW

File: src/uikit/Select/index.js [hunk 1]
```diff
@@ -1,24 +1,17 @@
...
-  box-sizing: border-box;
-  cursor: pointer;
-  padding: 5px;
-  top: 100%;
-`;
-
-const disabledDropdownOptionClassName = css`
-  color: lightgrey;
-`;
+export const optionDropdownWrapperClassName = css``;
```
Comment (jgnieuwhof) [inline src/uikit/Select/index.js]: :fire:

File: src/uikit/Select/index.js [hunk 2]
```diff
@@ -1,24 +1,17 @@
 import React from 'react';
 import Downshift from 'downshift';
 import { css } from 'emotion';
+import { truncate } from 'lodash';
 
-import downChevronIcon from '../assets/icon-chevron-down-grey.svg';
+import downChevronIcon from '../../assets/icon-chevron-down-grey.svg';
+import {
+  OptionDropdownWrapper,
+  DropDownLavelContent,
```
Comment (jgnieuwhof) [inline src/uikit/Select/index.js]: `DropDownLabelContent` ?

File: src/uikit/Select/ui.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  min-width: 100%;
+  z-index: 1;
+  border: 1px solid rgba(0, 0, 0, 0.05);
+  box-sizing: border-box;
+  cursor: pointer;
+  padding: 5px;
+  top: 100%;
+  right: ${({ align }) => (align === 'right' ? `0` : `auto`)};
+  left: ${({ align }) => (align === 'right' ? `auto` : `0`)};
+  ${({ className }) =>
```
Comment (jgnieuwhof) [inline src/uikit/Select/ui.js]: no need for this right? `styled` should pass `className` through as a prop & just apply it to the `div`

File: src/uikit/Select/ui.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import { css } from 'emotion';
+import styled from 'react-emotion';
+
+export const OptionDropdownWrapper = styled('div')`
+  position: absolute;
+  background: white;
+  min-width: 100%;
+  z-index: 1;
+  border: 1px solid rgba(0, 0, 0, 0.05);
```
Comment (jgnieuwhof) [inline src/uikit/Select/ui.js]: should be from theme's grayscale colors

File: src/uikit/Select/index.js [hunk 5]
```diff
@@ -29,16 +22,11 @@ export const DropDownOption = ({
...
+  <StyledDropDownOption
     key={item}
-    className={`
-      ${css`
-        cursor: pointer;
-        padding: 5px;
-      `}
-      ${itemClassName}
-      ${isItemDisabled({ item }) ? disabledDropdownOptionClassName : ''}`}
+    itemClassName={itemClassName}
```
Comment (jgnieuwhof) [inline src/uikit/Select/index.js]: rename to `className`

File: src/uikit/Select/ui.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+  margin-right: 12px;
+  transform: rotate(${({ isOpen }) => (isOpen ? 180 : 0)}deg);
+  transition: transform 0.2s;
+`;
+
+export const StyledDropDownOption = styled('div')`
+  cursor: pointer;
+  padding: 5px;
+  color: ${({ disabled }) => (disabled ? 'lightgrey' : 'auto')};
+  ${({ itemClassName }) =>
```
Comment (jgnieuwhof) [inline src/uikit/Select/ui.js]: if renamed to `className` as mentioned above no need to manually pass through

File: src/components/FileRepoSidebar/ReportsDownloadInput.js [hunk 7]
```diff
@@ -25,6 +25,7 @@ export default compose(withApi, withTheme)(({ api, projectId, theme, sqon, class
     graphqlField="participant"
     render={({ state }) => (
       <PillInputWithLoadingOptionsAndButton
+        selectedLabelTruncate={26}
```
Comment (jgnieuwhof) [inline src/components/FileRepoSidebar/ReportsDownloadInput.js]: is this a px width? or character count?

**Comment by hlminh2000** [comment]
@joneubank Addressed! :)

File: src/uikit/Select/index.js [hunk 8]
```diff
@@ -1,24 +1,16 @@
 import React from 'react';
 import Downshift from 'downshift';
+import { truncate } from 'lodash';
```
Comment (jgnieuwhof) [inline src/uikit/Select/index.js]: :fire:

File: src/uikit/Select/index.js [hunk 9]
```diff
@@ -1,24 +1,16 @@
...
-
-const disabledDropdownOptionClassName = css`
-  color: lightgrey;
-`;
+import downChevronIcon from '../../assets/icon-chevron-down-grey.svg';
+import {
+  OptionDropdownWrapper,
+  DropDownLabelContent,
+  ToggleImage,
+  StyledDropDownOption,
```
Comment (jgnieuwhof) [inline src/uikit/Select/index.js]: is there a reason this one's called `Styled` and the others aren't?

**Pull request merged**